### PR TITLE
fix: redirect from old link question. 

### LINKED ISSUE #72: Add redirect from old question links
Legacy app used links with the format `https://questions.wizeline.com/#/?questionId=1440` . With Remix, we now use the format `https://questions.wizeline.com/<questionId>/  `to get the detail of a question. This ticket is to add a redirect rule when you add ?questionId prop to the route. This could be achieved through the homepage's `loader`. This will allow support for old links in the Slack channels to work with the new app.

### PR DESCRIPTION
**Pull request by GeorginaFraire**
#### What does this PR do?
- Add support for the old questions links for the details : ` https://questions.wizeline.com/#/?questionId=1440 `

#### Where should the reviewer start?
-  /app/routes/index.jsx

#### How should this be manually tested?
1. Checkout for this branch `frontend/add-redict-to-legacy-url-question`
2. Run the project 
3. Login
4. Go to the next old links:
       http://localhost:3000/#/?questionId=1   - _redirect to the question detail_
       http://localhost:3000/#/?questionId=10  - _redirect to the question detail_
       http://localhost:3000/#/?questionId=2 - _redirect to the question detail_ 
       http://localhost:3000/#/?questionId=Hola - _stay on the homepage_
5. Replace the questionId with another id in your local environment. and verify can you can acces to the detail
      http://localhost:3000/#/?questionId={replace_id}

#### What are the relevant tickets?
(Link to issues, related PR, JIRA issues, etc.)
Closes #72

#### Screenshots

https://user-images.githubusercontent.com/97203617/201414367-d88b1a21-74c5-4cc8-8141-81c3dc91cc1c.mov

### CODE REVIEW

**Comment by albertcastaned** [comment]
Failed test has been occurring regularly, is now a ticket to fix #82 . I will re-run jobs to see if it passes by chance. If not, I would suggest commenting out this test until its fixed

File: app/routes/index.jsx [hunk 1]
```diff
@@ -76,13 +78,33 @@ export default function Index() {
...
+  useEffect(() => {
+    if(location.hash.includes('questionId')){
+      const hashQuestion = location.hash;
+      const questionId = hashQuestion.substring(hashQuestion.indexOf('questionId=') + 11, hashQuestion.length);
+      if(parseInt(questionId,10)){
+        navigate(`/questions/${questionId}`);
+      }
+      else {
+        setIsLoadign(false);
+        //To-Do Redirect to Not Found Question Page.
```
Comment (albertcastaned) [inline app/routes/index.jsx:100]: Think we will redirect to a 404 with #85 through the loader.

File: app/routes/index.jsx [hunk 2]
```diff
@@ -76,13 +78,33 @@ export default function Index() {
   const fetcher = useFetcher();
   const [shouldFetch, setShouldFetch] = useState(true);
   const [page, setPage] = useState(2);
+  const [isLoading, setIsLoadign] = useState(true);
```
Comment (albertcastaned) [inline app/routes/index.jsx:81]: Minor typo

File: app/routes/index.jsx (hunk 1 above)
Comment (GeorginaFraire) [reply app/routes/index.jsx]: I'll delete comment

File: app/routes/index.jsx (hunk 2 above)
Comment (GeorginaFraire) [reply app/routes/index.jsx]: Done ✅

**Review by albertcastaned — APPROVED** [review]
LGTM 👍

**Pull request merged**